code/modules/surgery/limb_augmentation.dm

```diff
@@ -16,7 +16,7 @@
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_selected)
-		if(affected.status & ORGAN_BROKEN) //The arm has to be in prime condition to augment it.
+		if(affected.status & ~(ORGAN_SPLINTED | ORGAN_SALVED)) //The arm has to be in prime condition to augment it.
```

Comment:
Might wanna clarify this new check logic in a comment as it's a less common syntax

Reply:
Added an updated comment on the new commit!